docs/endpoints/scan-results.md

```diff
@@ -24,35 +24,63 @@ The body must contain all collected data from the client's device.
   },
   "beaconLogs": [
     {
-      "major": 16,
       "UUID": "44c0ffee-988a-49dc-0bad-a55c0de2d1e4",
+      "major": 16,
+      "minor": 0,
       "timestamp": 1406804464.128,
+      "timezone": "UTC+01:00",
       "rssi": -60,
-      "id": 1,
-      "minor": 0,
       "beacon_id": 16
     }, {
-      "major": 16,
       "UUID": "44c0ffee-988a-49dc-0bad-a55c0de2d1e4",
+      "major": 16,
+      "minor": 0,
       "timestamp": 1406804474.128,
+      "timezone": "UTC+02:00",
       "rssi": -60,
-      "id": 2,
-      "minor": 0,
       "beacon_id": 16
     }
   ]
 }
 ```
 
+| Parameter               | Description                                              | Required |
+|-------------------------|----------------------------------------------------------|:--------:|
+| instance_ref            | The application reference as you get from your dashboard | yes      |
+| device.model            | The client's device model                                | yes      |
+| device.operating_system | The client's device model                                | yes      |
+| device.version          | The client's device OS version                           | yes      |
+| device.device_id        | The client's unique device id                            | yes      |
+| beaconLogs[].UUID       | The beacon UUID                                          | yes      |
+| beaconLogs[].major      | The beacon's major number                                | yes      |
+| beaconLogs[].minor      | The beacon's minor number                                | yes      |
+| beaconLogs[].timestamp  | Timestamp of result in seconds                           | yes      |
+| beaconLogs[].timezone   | The client's timezone in UTC format                      | no       |
```

Comment:
Just leaving this here for later if we ever decide to look on previous decisions; I still feel that the naming on this field is off. This should be timezoneOffset. Timezone (as seen in PHP) should be the actual timezone indicator in string form and must be one of the entries as seen in https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List in the tz column. That why (in PHP) 
See https://en.wikipedia.org/wiki/Time_zone#Time_zones_and_time_offsets